common/src/main/java/com/wynntils/models/items/annotators/game/IngredientAnnotator.java

```diff
@@ -25,15 +25,16 @@ public ItemAnnotation getAnnotation(ItemStack itemStack, String name) {
 
         String ingredientName = matcher.group(1);
         String tierColor = matcher.group(2);
-        int tier = Models.IngredientProfiles.getTierFromColorCode(tierColor);
 
-        IngredientProfile ingredientProfile = Models.IngredientProfiles.getIngredient(ingredientName);
-        if (ingredientProfile == null) return null;
-        if (ingredientProfile.getTier().getTierInt() != tier) {
+        int tier = Models.Ingredient.getTierFromColorCode(tierColor);
+        IngredientInfo ingredientInfo = Models.Ingredient.getIngredientInfoFromName(ingredientName);
+        if (ingredientInfo == null) return null;
+
+        if (ingredientInfo.tier() != tier) {
             WynntilsMod.warn("Incorrect tier in ingredient database: " + ingredientName + " is " + tier);
             return null;
```

Comment:
I did remove this return null though, it is unnecessary. We can warn for the inconsistency, but we have something to go with, it's better than nothing.